src/api/index.ts

```diff
@@ -148,6 +148,7 @@ export default class ApiService {
     if (response?.success) {
       return response.data;
     }
+    return null;
```

Comment:
has to be null rather than undefined because next hates _undefined_ when it comes to passing props to a page 

https://github.com/Coding-Coach/find-a-mentor/blob/2b07d569fbd1228f592d2b71f341bcfc05f020e1/pages/u/%5Bid%5D.tsx#L29-L35